CHANGES.rst

```diff
@@ -0,0 +1,6 @@
+0.1.1
```

Comment:
Any reason why this value is used as the first version?
Seems odd to have a patch number when there is not even an initial `0.1.0`.

Reply:
No, good catch I'll change it to 0.1.0